apps/build-onchain-apps/src/components/buy-me-coffee/FormBuyCoffee.tsx

```diff
@@ -75,6 +76,18 @@ function FormBuyCoffee({ onComplete }: FormBuyCoffeeProps) {
     [setMessage],
   );
 
+  if (contract.status === 'notConnected') {
```

Comment:
If those are common for all contracts, I wonder if we should expose something that makes this logic even simpler.... I guess that can be a second step. Let's probably explore what's the full experience and than we can simplify even more.